spring-cloud-stream/src/main/java/org/springframework/cloud/stream/aggregate/AggregateApplicationBuilder.java

```diff
@@ -49,6 +49,10 @@
 
 	private ConfigurableApplicationContext parentContext;
 
+	public AggregateApplicationBuilder(String... args) {
```

Comment:
Since the issue says no-arg constructor, does this PR require changes to include the constructor public AggregateApplicationBuilder() as well?